uaclient/apt_news.py

```diff
@@ -263,8 +263,17 @@ def local_apt_news(cfg: UAConfig) -> Optional[str]:
 
 
 def format_news_for_apt_update(news: str) -> str:
+    lines = news.split("\n")
+
+    if system.get_release_info().release >= "24.10":
+        result = "\nAPT news:"
```

Comment:
One test failed: features/apt_messages.feature:755 with

```
      Assertion Failed: Expected to find exactly:
        Reading package lists...
        Building dependency tree...
        Reading state information...
        Calculating upgrade...

        APT news:
          one

        Summary:
          Upgrading: 0, Installing: 0, Removing: 0, Not Upgrading: 0
      But got:
        Reading package lists...
        Building dependency tree...
        Reading state information...
        Calculating upgrade...

        APT news:  one

        Summary:
          Upgrading: 0, Installing: 0, Removing: 0, Not Upgrading: 0

```

I think this will solve it
```suggestion
        result = "\nAPT news:\n"
```

Reply:
I was just running these locally, haha, thanks for the find!